app/controllers/ops_controller/settings/common.rb

```diff
@@ -173,8 +173,97 @@ def smartproxy_affinity_field_changed
     end
   end
 
+  def pglogical_subscriptions_form_fields
+    replication_type = MiqRegion.replication_type
+    subscriptions = replication_type == :global ? PglogicalSubscription.all : []
+    subscriptions = get_subscriptions_array(subscriptions) unless subscriptions.empty?
+    render :json => {
+      :replication_type => replication_type,
+      :subscriptions    => subscriptions
+    }
+  end
+
+  def pglogical_save_subscriptions
+    replication_type = valid_replication_type
+    if replication_type == :global
+      MiqRegion.replication_type = replication_type
+      subscriptions_to_save = []
+      params[:subscriptions].each do |_, subscription|
+        sub = subscription['id'] ? PglogicalSubscription.find(subscription['id']) : PglogicalSubscription.new
+        sub.dbname   = subscription['dbname']
+        sub.host     = subscription['host']
+        sub.user     = subscription['user']
+        sub.password = subscription['password'] == '●●●●●●●●' ? nil : subscription['password']
+        sub.port     = subscription['port']
+        if sub.id && sub['remove'] == "true"
```

Comment:
This still needs to be `subscription['remove'] as `sub` is the `PglogicalSubscription` object.